app/builders/account_builder.rb

```diff
@@ -79,4 +82,20 @@ def create_user
     @user.confirm if @confirmed
     @user.save!
   end
+
+  def domain_blocked?(email)
+    domain = email.split('@').last
+    # loop over blocked_domains and do a regex match
```

Comment:
Do we need this comment?

Reply:
Oops, added it for copilot :P 
removing it